src/components/Helper.js

```diff
@@ -0,0 +1,27 @@
+/* eslint no-eval: 0 */
+const getRandomOperator = (quiz) => {
+  const opIndex = Math.floor(Math.random() * quiz.operatorsVal.length);
+  return quiz.operatorsVal[opIndex];
+};
+const getRandomNumber = (quiz) => {
```

Comment:
Try to make helpers more generic. Instead of passing quiz, pass the min and max value. 

Same for the other helper methods. If they are not directly linked to a quiz, make the params independent of quiz as well.

Reply:
Done